token/provider/github.go

```diff
@@ -0,0 +1,23 @@
+package provider
+
+import (
+	"fmt"
+
+	"github.com/fabric8-services/fabric8-auth/client"
+
+	"golang.org/x/oauth2"
+	"golang.org/x/oauth2/github"
+)
+
+type GitHubConfig struct {
+	oauth2.Config
+}
+
+func NewGitHubConfig(clientID string, clientSecret string, authURL string) *GitHubConfig {
+	provider := &GitHubConfig{}
+	provider.ClientID = clientID
+	provider.ClientSecret = clientSecret
+	provider.Endpoint = github.Endpoint
+	provider.RedirectURL = fmt.Sprintf("%s%s", authURL, client.CallbackTokenPath())
```

Comment:
curious, is this the only use-case we have for the `client/...` package?

Reply:
Correct.